test/Mono.Linker.Tests/TestCasesRunner/AssemblyChecker.cs

```diff
@@ -615,6 +615,12 @@ protected virtual IEnumerable<string> FilterLinkedAttributes (ICustomAttributePr
 					case "System.Diagnostics.DebuggerBrowsableAttribute":
 						continue;
 
+#if NETCOREAPP
```

Comment:
IIRC the issue is that this will prevent looking for the attribute in the linked assembly, but testcases that specify it as a KeptAttribute were still failing because it was included in GetExpectedAttributes.

Reply:
I believe the tests that specify it as kept are the ones the explicitly use roslyn.  The tests don't actually care about the DebuggableAttribute, they just needed to assert it because roslyn was used.  You would remove the `[Kept(DebuggableAttribute)]` from tests.

Reply:
Gotcha - I'll filter that attribute on mono as well and get rid of the attribute everywhere.

Reply:
Just noting that these testcases won't be checking for the attribute even though they used to (even with mcs):
- Assemblywithdefaultsymbols
- AssemblyWithDefaultSymbolsAndSymbolLinkingEnabled
- ReferencesWithMixedSymbolTypes
- CanCompileTestCaseWithDebugPdbs
- CanCompileTestCaseWithMsc

I think that's fine since they probably didn't care about the attribute either. LMK if you think otherwise.

Reply:
👍